modules/vaos/app/controllers/vaos/v2/appointments_controller.rb

```diff
@@ -21,14 +21,16 @@ class AppointmentsController < VAOS::BaseController
       COMMENT = 'comment'
 
       def index
-        appointments[:data].each do |appt|
+        merged_appointments = appointments_service.merge_appointments(eps_appointments, appointments)
+
+        merged_appointments.each do |appt|
           set_facility_error_msg(appt) if include_index_params[:facilities]
           scrape_appt_comments_and_log_details(appt, index_method_logging_name, PAP_COMPLIANCE_TELE)
           log_appt_creation_time(appt)
         end
 
         serializer = VAOS::V2::VAOSSerializer.new
-        serialized = serializer.serialize(appointments[:data], 'appointments')
+        serialized = serializer.serialize(merged_appointments, 'appointments')
 
         if !appointments[:meta][:failures]&.empty?
```

Comment:
You'd need to update other code within this method that refers to original appointments array as well

Reply:
Note to self, appointments isn't called at all here